CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 ## Unversioned
 
+- Major: Added "Channel Filters". See https://wiki.chatterino.com/Filters/ for how they work or how to configure them. (#1748, #2083)
+- Major: Added Streamer Mode configuration (under `Settings -> General`), where you can select which features of Chatterino should behave differently when you are in Streamer Mode. (#2001)
```

Comment:
Please undo the re-ordering of these two entries

Reply:
Resolved in 6bd19a6